.swiftformat

```diff
@@ -1,4 +1,4 @@
---exclude Sources/MongoSwift/MongoSwiftVersion.swift
+--exclude Sources/MongoSwift/MongoSwiftVersion.swift,Sources/MongoSwiftSync/*
```

Comment:
I had to disable this because otherwise the formatter automatically replaces each parameter name with an underscore as they are all unused. I opened [SWIFT-680](https://jira.mongodb.org/browse/SWIFT-680) about reenabling it so we don't forget.